src/ir/type-updating.cpp

```diff
@@ -342,6 +342,7 @@ Type GlobalTypeRewriter::getTempType(Type type) {
   if (type.isRef()) {
     auto heapType = type.getHeapType();
     if (auto it = typeIndices.find(heapType); it != typeIndices.end()) {
+      // TODO: Handle exactness.
```

Comment:
The fuzzer does not seem to trip over this (yet), so I left it unfixed for now. Eventually I expect us to trip over this bug, at which point we will be able to reduce the reproducer to a test case and ensure we have test coverage.